director/director_api.go

```diff
@@ -33,3 +49,78 @@ func ListNamespacesFromOrigins() []NamespaceAd {
 	}
 	return namespaces
 }
+
+func LoadDirectorPublicKey() (*jwk.Key, error) {
+	directorUrlStr := param.Federation_DirectorUrl.GetString()
+	if len(directorUrlStr) == 0 {
+		return nil, errors.Errorf("Director URL is unset; Can't load director's public key")
+	}
+	log.Debugln("Director URL:", directorUrlStr)
+	directorUrl, err := url.Parse(directorUrlStr)
+	if err != nil {
+		return nil, errors.Wrap(err, fmt.Sprintln("Invalid director URL:", directorUrlStr))
+	}
+	directorUrl.Scheme = "https"
+
+	discoveryUrl, _ := url.Parse(directorUrl.String())
+	discoveryUrl.Path = path.Join(".well-known/pelican-configuration", discoveryUrl.Path)
```

Comment:
In this case, we're probably better off hardcoding the separator because we're dealing with a URL path, which is universal -- if you were to run this on Windows, `path.Join()` would join the paths with `\` instead of `/`